docs/tools/lsp-smart-contracts/contracts-abi.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 2
+---
+
+# Contracts ABI
+
+You can import the [LUKSO smart contracts](../../standards/smart-contracts/introduction.md) ABIs from the `/artifacts` folder:
+
+```javascript
+import LSP0ERC725Account from '@lukso/lsp-smart-contracts/artifacts/LSP0ERC725Account.json';
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
+import LSP9Vault from '@lukso/lsp-smart-contracts/artifacts/LSP9Vault.json';
+// etc.
```

Comment:
Are these imports going to return you the artifacts or the abis? because If it returns the artifacts, you will need to do `.abi` on the constants in order to get the abis. wdyt @YamenMerhi @YamenMerhi ?

Reply:
Maybe putting an example of creating a contract instance might help?

(pseudo code)
```js
const account = new web3.contract(LSP0ERC725Account.abi, "<contract-address>")
```